### ISSUE TITLE
plugin is unstable

### ISSUE THREAD

**Issue by hannesdelbeke** [MamoruDS/obsidian-open-link-with#16]
currently, when clicking a URL. nothing happens

the dev console shows these errors:
```javascript
plugin:obsidian-open-link-with:761 [open-link-with] fake-el with "fake_extlink" not found
```
```javascript
VM168 plugin:obsidian-open-link-with:761 [open-link-with] fake-el with "fake_extlink" not found
```

the plugin stopped working as expected. changing plugin settings appears to have broken it and i can't change it back without disabling the whole plugin.

open external links: system-default
removed any modifier bindings
 
 
<img width="613" alt="image" src="https://user-images.githubusercontent.com/3758308/208208941-4849bb0e-3c1c-410a-b570-748a79acbabc.png">

**Comment by MamoruDS** [comment]
Hey there! Sorry for the late reply. Thanks for reporting the issue. I rewrote the click handle processes and fixed the plugin unload handling, so these issues should be sorted now. The plugin should be stable for Obsidian client versions `>=1.1`.  Thank you for using the plugin, and please let me know if there are any other problems.

**Comment by MamoruDS** [comment]
If you are encountering this problem in live preview mode, it is likely a click behavior issue. This should be resolved in the latest release (`0.1.9`), but please let me know if you are still experiencing the problem.
